### PR TITLE
Move documentation build scripts and fix link to github tests

### PR DESCRIPTION
**Pull request by nickjalbert**
* Found the link on the github testing badge was broken so fixed that in b1ff04a

agentosbadge

* Decided to knock out https://github.com/agentos-project/agentos/issues/48 while I was in there (although, I'll hold off on fixing "good first issues" in the future)

### CODE REVIEW

File: scripts/build_docs.py [hunk 1]
```diff
@@ -3,21 +3,25 @@
...
 
 from importlib.machinery import SourceFileLoader
 import os
 from subprocess import Popen
 
-docs_dir = os.path.dirname(os.path.abspath(__file__))
-version_file = os.path.join(docs_dir, os.pardir, "agentos", "version.py")
+scripts_dir = os.path.dirname(os.path.abspath(__file__))
+version_file = os.path.join(scripts_dir, os.pardir, "agentos", "version.py")
 loaded = SourceFileLoader("agentos.version", version_file).load_module()
```
Comment (andyk) [inline scripts/build_docs.py:15]: Maybe it would be cleaner if we used `agentos.version` here?

File: scripts/build_docs.py [hunk 2]
```diff
@@ -3,21 +3,25 @@
...
+  $ python scripts/build_docs.py
 """
 
 from importlib.machinery import SourceFileLoader
 import os
 from subprocess import Popen
 
-docs_dir = os.path.dirname(os.path.abspath(__file__))
-version_file = os.path.join(docs_dir, os.pardir, "agentos", "version.py")
+scripts_dir = os.path.dirname(os.path.abspath(__file__))
```
Comment (andyk) [inline scripts/build_docs.py:13]: Do you think it's worth moving the creation of these common path variables to `shared.py`?

**Review by andyk — APPROVED** [review]
LGTM as is, though I left comments with a couple of ideas that you could implement if you agree or like  them, but not necessary at all.

File: scripts/build_docs.py (hunk 2 above)
Comment (nickjalbert) [reply scripts/build_docs.py:13]: sounds good!

File: scripts/build_docs.py (hunk 1 above)
Comment (nickjalbert) [reply scripts/build_docs.py:15]: I'll play around with that.   Should I just be able to `import agentos`  directly given the state of the requirements?

Comment (nickjalbert) [reply scripts/build_docs.py:15]: ah looks like `agentos.__version__` works.  Is that what you were thinking?

Comment (andyk) [reply scripts/build_docs.py]: Ha, yes, that's what I was thinking.

**Comment by nickjalbert** [comment]
comments addressed and tests passing.  merging!

**Pull request merged**